src/leap/mail/imap/service/imap-server.tac

```diff
@@ -144,6 +124,8 @@ km_kwargs = {
 }
 keymanager = KeyManager(*km_args, **km_kwargs)
 
+# XXX Do we need to wait until keymanager is properly initialized?
+
```

Comment:
There is no need to wait for keymanager, as soon as the constructor returns you can use the object without problems, the only exception is the method `get_all_keys` that might fail if the initialization is not done, but this method is not used here.